lib/matplotlib/tests/test_ticker.py

```diff
@@ -1594,6 +1594,79 @@ def test_engformatter_usetex_useMathText():
         assert x_tick_label_text == ['$0$', '$500$', '$1$ k']
 
 
+@pytest.mark.parametrize(
+    'oom_center, oom_noise, oom_center_desired, oom_noise_desired', [
```

Comment:
I would write `oom_center` and `oom_noise` here as their actual values, instead of `log10` of them. Then it should (hopefully) be clear where in the relative order of magnitude should be. You can write them grouped in 3s such as:
```
        (271_490_000_000, 10, 9, 0),
        (27_149_000_000_000, 10_000_000, 12, 6),
        ...
```

Reply:
> I would write `oom_center` and `oom_noise` here as their actual values, instead of `log10` of them (and modify the test to not do `10**oom_{center,noise}`.)
> 
> Then it should (hopefully) be clear where in the relative order of magnitude should be. You can write them grouped in 3s such as:
> 
> ```
>         (271_490_000_000, 10, 9, 0),
>         (27_149_000_000_000, 10_000_000, 12, 6),
>         ...
> ```

Hmm but then isn't it a bit weird how `oom_` means something different when it is the first 2 values v.s the second? It is much more intuitive to me how all of these variables are in the same logarithmic scale, and it would be easier to maintain these tests IMO like this (at least for me).

Reply:
Well yes, you would remove the `oom_` prefix on those ones. The problem with using order-of-magnitude to me is that it seems a bit circular, just from looking at the parametrization.